daikoku/app/storage/drivers/postgres/Helper.scala

```diff
@@ -0,0 +1,134 @@
+package storage.drivers.postgres
+
+import jdk.jshell.spi.ExecutionControl.NotImplementedException
```

Comment:
Yep, I think we need to redefine this one ;)

Reply:
This class is only present in jdk 13+. I changed this class for `UnsupportedOperationException` class, already present in version 8.